src/views/illness/illness-start-date.njk

```diff
@@ -0,0 +1,111 @@
+{% extends 'layout.njk' %}
+
+{% from 'govuk/components/back-link/macro.njk' import govukBackLink %}
+{% from 'govuk/components/error-summary/macro.njk' import govukErrorSummary %}
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+{% from 'govuk/components/date-input/macro.njk' import govukDateInput %}
+
+{% block pageTitle %}
+   Appeal due to ill health: Tell us when the illness started
```

Comment:
Is this page title stated in the ticket? Tell has a capital T even though its not starting a sentence, is that how its meant to be?

Reply:
I've taken this from the prototype